### ISSUE TITLE
[OpenBMC] Python error display for interface with multiple IPs crashes 

### ISSUE THREAD

**Issue by gurevichmark** [xcat2/xcat-core#6230]
In Python, the code for processing `rspconfig` command is in `xcat-core/xCAT-openbmc-py/lib/python/agent/hwct/openbmc_client.py` file. Function `get_netinfo`. While processing network related flag, like `rspconfig ntpservers` or `rspconfig hostname` if an interface with multiple IPs is detected, an error message is supposed to be displayed.

```
if 'ip' in netinfo[nicid]:
                        msg = "%s: Another valid ip %s found." % (node, v["Address"])
                        self._print_record_log(msg, 'get_netinfo')
                        del netinfo[nicid]
                        netinfo['error'] = 'Interfaces with multiple IP addresses are not supported'
                        break
```

However, the `node` variable does not exist and crashes with `NameError: global name 'node' is not defined` error logged into `/var/log/xcat/agent.log`

Perl code works fine, the problem is with Python code only.

This problem can be recreated on `briggs01`. There node `mid05tor12cn03` works ok, but `mid05tor12cn05` does not.

The  `mid05tor12cn05` has an interface with multiple IPs (`172.11.139.4` and `172.11.139.5`):
```
[root@briggs01 ~]# curl -k -c cjar -b cjar -X GET -H "Content-Type: applicationjson" https://172.11.139.5/xyz/openbmc_project/network/enumerate
{
  "data": {
    "/xyz/openbmc_project/network/config": {
      "DefaultGateway": "172.11.253.27",
      "HostName": "mid05tor12cn05-bmc-test"
    },
    "/xyz/openbmc_project/network/config/dhcp": {
      "DNSEnabled": true,
      "HostNameEnabled": true,
      "NTPEnabled": true,
      "SendHostNameEnabled": true
    },
    "/xyz/openbmc_project/network/eth0": {
      "AutoNeg": false,
      "DHCPEnabled": false,
      "DomainName": [],
      "InterfaceName": "eth0",
      "MACAddress": "70:e2:84:14:28:0d",
      "NTPServers": [],
      "Nameservers": [],
      "Speed": 0
    },
    "/xyz/openbmc_project/network/eth0/ipv6/28f8222a": {
      "Address": "fe80::72e2:84ff:fe14:280d",
      "Gateway": "",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.LinkLocal",
      "PrefixLength": 64,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv6"
    },
    "/xyz/openbmc_project/network/eth0_11": {
      "AutoNeg": false,
      "DHCPEnabled": false,
      "DomainName": [],
      "Id": 11,
      "InterfaceName": "eth0.11",
      "MACAddress": "70:e2:84:14:28:0d",
      "NTPServers": [],
      "Nameservers": [],
      "Speed": 0
    },
    "/xyz/openbmc_project/network/eth0_11/ipv4/6dc1614b": {
      "Address": "172.11.139.5",
      "Gateway": "172.11.253.27",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.Static",
      "PrefixLength": 16,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/eth0_11/ipv4/ac98f1aa": {
      "Address": "172.11.139.4",
      "Gateway": "172.11.253.27",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.Static",
      "PrefixLength": 16,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/eth0_11/ipv6/28f8222a": {
      "Address": "fe80::72e2:84ff:fe14:280d",
      "Gateway": "",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.LinkLocal",
      "PrefixLength": 64,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv6"
    },
    "/xyz/openbmc_project/network/host0/intf": {
      "MACAddress": "00:00:00:00:00:00"
    },
    "/xyz/openbmc_project/network/host0/intf/addr": {
      "Address": "0.0.0.0",
      "Gateway": "0.0.0.0",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.Static",
      "PrefixLength": 0,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/snmp/manager": {}
  },
  "message": "200 OK",
  "status": "ok"
}[root@briggs01 ~]#
```

The `mid05tor12cn03` also has an interface with multiple IPs (`169.254.239.67` and `172.12.253.101`), But since one of them starts with `169.254` is it ignored and processing is allowed to continue:
```
[root@briggs01 ~]#  curl -k -c cjar -b cjar -X GET -H "Content-Type: applicatio/json" https://172.12.253.101/xyz/openbmc_project/network/enumerate
{
  "data": {
    "/xyz/openbmc_project/network/config": {
      "DefaultGateway": "172.12.253.27",
      "HostName": "witherspoon"
    },
    "/xyz/openbmc_project/network/config/dhcp": {
      "DNSEnabled": true,
      "HostNameEnabled": true,
      "NTPEnabled": true,
      "SendHostNameEnabled": true
    },
    "/xyz/openbmc_project/network/eth0": {
      "AutoNeg": false,
      "DHCPEnabled": true,
      "DomainName": [],
      "InterfaceName": "eth0",
      "MACAddress": "70:e2:84:14:29:1b",
      "NTPServers": [],
      "Nameservers": [],
      "Speed": 0
    },
    "/xyz/openbmc_project/network/eth0/ipv4/7fcc9b02": {
      "Address": "169.254.239.67",
      "Gateway": "0.0.0.0",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.DHCP",
      "PrefixLength": 16,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/eth0/ipv4/97d9b49": {
      "Address": "172.12.253.101",
      "Gateway": "0.0.0.0",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.DHCP",
      "PrefixLength": 16,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/eth0/ipv6/524d0f5b": {
      "Address": "fe80::72e2:84ff:fe14:291b",
      "Gateway": "",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.DHCP",
      "PrefixLength": 64,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv6"
    },
    "/xyz/openbmc_project/network/host0/intf": {
      "MACAddress": "00:00:00:00:00:00"
    },
    "/xyz/openbmc_project/network/host0/intf/addr": {
      "Address": "0.0.0.0",
      "Gateway": "0.0.0.0",
      "Origin": "xyz.openbmc_project.Network.IP.AddressOrigin.Static",
      "PrefixLength": 0,
      "Type": "xyz.openbmc_project.Network.IP.Protocol.IPv4"
    },
    "/xyz/openbmc_project/network/snmp/manager": {}
  },
  "message": "200 OK",
  "status": "ok"
}[root@briggs01 ~]#
```

**Comment by xuweibj** [comment]
I think modified code to ``msg = "Another valid ip %s found." % (v["Address"])`` is OK. But we did not support multiple IPs, will not get information like ip or ntpservers.

**Comment by gurevichmark** [comment]
Fixed by PR #6237
